devtools/maven/pom.xml

```diff
@@ -142,6 +142,29 @@
             <artifactId>jProcesses</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
It doesn't look like you're using this plug-in?

Reply:
It's used in the test to stop the running processed in a "reliable" way:
https://github.com/quarkusio/quarkus/blob/master/devtools/maven/src/test/java/io/quarkus/maven/it/verifier/RunningInvoker.java#L52